### DISCUSSION TITLE
TypeError: argument should be a bytes-like object or ASCII string, not 'list'

### DISCUSSION THREAD

**Discussion by abrahamU** [langgenius/dify#3338] [Help]
Hello Dify team

I tried to upload a doc file for knowledge base, after the preprocess, it returned with an error
TypeError: argument should be a bytes-like object or ASCII string, not 'list'

but I saw some content is already inside, this error happens on both self hosted server and cloud server

could you give me a hint on how to solve it?
thanks.

![image](https://github.com/langgenius/dify/assets/17700411/39bf3eff-1ac9-4937-a4a5-bc4d4b604bd2)

**Comment by xingmolu** [comment]
@abrahamU Has this problem been solved?

**Comment by Nobellcq** [comment]
same question, anyone can help

**Comment by cumtsd** [comment]
同样的问题，请问解决了吗
